include/deal.II/fe/fe_series.h

```diff
@@ -116,20 +116,13 @@ namespace FESeries
     /**
      * hp::FECollection for which transformation matrices will be calculated.
      */
-    SmartPointer<const hp::FECollection<dim>> fe_collection;
+    SmartPointer<const hp::FECollection<dim, spacedim>> fe_collection;
 
     /**
      * hp::QCollection used in calculation of transformation matrices.
      */
     SmartPointer<const hp::QCollection<dim>> q_collection;
 
-    /**
-     * Ensure that the transformation matrix for FiniteElement index
-     * @p fe_index is calculated. If not, calculate it.
-     */
-    void
-    ensure_existence(const unsigned int fe_index);
```

Comment:
@davydden Can I just confirm: Was this always meant to be an internal function, not to be called by the user? If not, then I think that it should be deprecated instead of removed.

Reply:
> Was this always meant to be an internal function, not to be called by the user?

i presume you did not notice it sitting in `private:` part ;-) so yes, it's internal

Reply:
Whoops, no I did not notice that :-) Thanks!